packages/volto-slate/news/7180.internal

```diff
@@ -0,0 +1 @@
+Optimize and lint ObjectByTypeWidget inside the volto-slate package. @silviubogan
```

Comment:
The news goes into `volto-slate` history, so you don't need to say it in a change log.

```suggestion
Optimize and lint `ObjectByTypeWidget`. @silviubogan
```